### PR TITLE
Update READMEs and documentations

### LINKED ISSUE #1407: Missing quality level for Stackdriver Trace(Google Cloud Trace)
It seems like README.md misses quality level indication for Stackdriver Trace. Is it intentionally missing?

### PR DESCRIPTION
**Pull request by hxiong388**
Went through the Stackdriver libraries and made another round of updates on the READMEs and documentations.

[close #1407 ]

### CODE REVIEW

File: google-cloud-trace/lib/google/cloud/trace/trace_record.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ module Trace
       #   env = my_get_rack_environment
       #   trace_context = Stackdriver::Core::TraceContext.parse_rack_env env
       #
-      #   trace = Google::Cloud::Trace.new "my-project", trace_context
+      #   trace = Google::Cloud::TraceRecord.new "my-project", trace_context
```
Comment (dazuma) [inline google-cloud-trace/lib/google/cloud/trace/trace_record.rb]: Google::Cloud::Trace::TraceRecord

File: google-cloud-error_reporting/README.md [hunk 2]
```diff
@@ -58,14 +59,17 @@ end
...
 
-This library also provides a built-in Railtie for Ruby on Rails integration. To do this, simply add this line to config/application.rb:
+This library also provides a built-in Railtie for Ruby on Rails integration. To
+ do this, simply add this line to config/application.rb:
 ```ruby
 require "google/cloud/error_reporting/rails"
 ```
 
-Alternatively, check out [stackdriver](../stackdriver) gem, which includes this library and enables the Railtie by default.
+Alternatively, check out [stackdriver](../stackdriver) gem, which includes this
```
Comment (quartzmo) [inline google-cloud-error_reporting/README.md]: check out **the** [stackdriver

File: google-cloud-error_reporting/README.md [hunk 3]
```diff
@@ -58,14 +59,17 @@ end
...
 require "google/cloud/error_reporting/rails"
 ```
 
-Alternatively, check out [stackdriver](../stackdriver) gem, which includes this library and enables the Railtie by default.
+Alternatively, check out [stackdriver](../stackdriver) gem, which includes this 
+library and enables the Railtie by default.
 
-For Rack integration and more examples, see [Instrumentation Guide](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-error_reporting/guides/instrumentation).
+For Rack integration and more examples, see 
+[Instrumentation Guide](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-error_reporting/guides/instrumentation).
```
Comment (quartzmo) [inline google-cloud-error_reporting/README.md]: see **the** [Instrumentation

File: google-cloud-trace/README.md [hunk 4]
```diff
@@ -1,45 +1,94 @@
...
 
-    $ gcloud auth login
+This library also provides a built-in Railtie for Ruby on Rails integration. To
+ do this, simply add this line to config/application.rb:
+```ruby
+require "google/cloud/trace/rails"
+```
 
-Please see [gcloud beta auth application-default login][] document for the difference between these commands.
+Alternatively, check out [stackdriver](../stackdriver) gem, which includes this
```
Comment (quartzmo) [inline google-cloud-trace/README.md]: check out **the** [stackdriver

File: google-cloud-trace/README.md [hunk 5]
```diff
@@ -1,45 +1,94 @@
...
+```
 
-Please see [gcloud beta auth application-default login][] document for the difference between these commands.
+Alternatively, check out [stackdriver](../stackdriver) gem, which includes this 
+library and enables the Railtie by default.
 
-[Google Cloud SDK]: https://cloud.google.com/sdk/
-[gcloud beta auth application-default login]: https://cloud.google.com/sdk/gcloud/reference/beta/auth/application-default/login
+For Rack integration and more examples, see 
+[Instrumentation Guide](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-error_reporting/guides/instrumentation).
```
Comment (quartzmo) [inline google-cloud-trace/README.md]: see **the** [Instrumentation

File: google-cloud-trace/lib/google/cloud/trace/trace_record.rb (hunk 1 above)
Comment (hxiong388) [inline google-cloud-trace/lib/google/cloud/trace/trace_record.rb]: Lol, fixed.

**Pull request merged**